Integrations/integration-Netskope.yml

```diff
@@ -0,0 +1,319 @@
+            var dlp_file =  arrayItem.dlp_file;
+            var hostname =  arrayItem.hostname;
+            var policy =  arrayItem.policy;
+            retArray.push({"App" : app, "Timestamp" : timestamp, "DLPProfile" : dlp_profile, "DLPFile" : dlp_file, "Hostname" : hostname, "Policy" : policy})
+        });
+        var ec = {
+            "Netskope.Alerts" : retArray
+        };
+        return {
+            Type: entryTypes.note,
+            ContentsFormat: formats.json,
+            Contents: retArray,
+            ReadableContensFormat: formats.markdown,
+            HumanReadable: tableToMarkdown('Netskope alerts', retArray),
+            EntryContext: ec
+        };
+    }
+
+    function getEvents() {
+        var queryArgs = {
+            token: TOKEN,
+            type: encodeURIComponent(args.type)
+        };
+        if (!args.timeperiod && !args.query) {
+            queryArgs.starttime = args.starttime;
+            queryArgs.endtime = args.endtime;
+        } else if (!args.timeperiod && !args.query) {
+            queryArgs.starttime = args.starttime;
+            queryArgs.endtime = args.endtime;
+            queryArgs.query = encodeURIComponent(args.query);
+        } else if (!args.query) {
+            if (!args.timeperiod) {
+                queryArgs.starttime = args.starttime;
+                queryArgs.endtime = args.endtime;
+            } else {
+                queryArgs.timeperiod = args.timeperiod;
+            }
+        } else if (args.query && args.timeperiod) {
+            queryArgs.timeperiod = args.timeperiod;
+            queryArgs.query = encodeURIComponent(args.query);
+        } else {
+            throw 'Not given enough arguments to filter events by.'
+        }
+
+        var cmdUrl = 'events' + encodeToURLQuery(queryArgs);
+        var result = sendRequest('GET', cmdUrl);
+        var retArray = [];
+        result.data.forEach(function(arrayItem) {
+            var app =  arrayItem.app;
+            var timestamp = normalizeTimestamp(arrayItem.timestamp);
+            var activity =  arrayItem.activity;
+            var object =  arrayItem.object;
+            var hostname =  arrayItem.hostname;
+            var category =  arrayItem.category;
+            var device_classification =  arrayItem.device_classification;
+            var user =  arrayItem.user;
+            var from_user =  arrayItem.from_user;
+            var to_user =  arrayItem.to_user;
+            var srcip =  arrayItem.srcip;
+            var access_method =  arrayItem.access_method;
+            var url =  arrayItem.url;
+            retArray.push({"App" : app, "Timestamp" : timestamp, "Activity" : activity, "Object" : object, "Hostname" : hostname, "AppCategory" : category, "DeviceClassification" : device_classification, "User" : user, "FromUser" : from_user, "ToUser" : to_user, "SourceIP" : srcip, "AccessMethod" : access_method, "URL" : url});
+        });
+        var ec = {
+            "Netskope.Events" : retArray

```

Comment:
there should be event.id, so please add it and print it as the first column,also make sure the context merged by this id `Netskope.Events(val.Id==obj.Id)`